content/language/word-list/man-in-the-middle.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Man-in-the-middle attack
+---
+
+**Reasoning:** 
+
+When an attacker secretly intercepts and relays communications between two systems or people who believe that they are communicating directly with each other.
+
+Gendered language that can perpetuate bias and stereotypes.
+Implies that women do not have the skills to perpetrate this type of hacking.
+
+
+**Recommendation:** Adopt immediately.
+
+Recommended replacement terms
+-	Interceptor attack
+-	Intermediary attack
```

Comment:
Could I suggest some additional alternatives? I have recently seen the following being used "machine in the middle" and "attacker in the middle"

Reply:
Thanks, @glawrence , for your input. Below is the discussion of other terms that we considered, including machine in the middle. I don't remember whether attacker in the middle was discussed. 
If we add more terms at this stage, I think we'll have to stop this process and resubmit the term and its replacements for another public review period--is that correct @LarryKunz and @quaid ? 
Discussion of other terms:
What is a suitable replacement term? Proposals include:

- Machine-in-the-middle attack  - Some discussion seen that this might not be great, as it would also be shortened to “MITM attack” just as “man-in-the-middle attack” is, and won’t change the cultural use of the term.
- Person-in-the-middle attack
- On-path attack – Not clear about the “in the middle” part
- Interceptor attack – Like this, as it is nice and short and captures the aspect of data interception.

Reply:
That's fine @cbiagiBMC I just noticed this on Slack and also had seen conversation elsewhere, so was just joining things up

Reply:
> Thanks, @glawrence , for your input. Below is the discussion of other terms that we considered, including machine in the middle. I don't remember whether attacker in the middle was discussed. 
> If we add more terms at this stage, I think we'll have to stop this process and resubmit the term and its replacements for another public review period--is that correct @LarryKunz and @quaid ? 

Yes, we are working to publish all the terms that have been through the full reviews. That does not mean that they are perfect and cannot be added to. In fact, we can expect further interest in changes and improvements. We may end up building out more information for the choices made if the explanations are not complete enough, such as might be with here to include e.g. why we didn't include machine in the middle.

As part of getting this published, we need to include a clear change process for people who have questions, concerns, objections, etc.

@glawrence  +1 for jumping in and being aware, thanks.